### PR TITLE
feat: #SA-8 adds rule for reviewers

### PR DESCRIPTION
**Pull request by adrian-livefront**

### CODE REVIEW

**Comment by dperez37** [comment]
So this needs to be in the repo, it is not a Dependabot configuration?

**Review by dperez37 — APPROVED** [review]
LGTM, can you squash it before merging though

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+  - package-ecosystem: "bundler"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```
Comment (byencho) [inline .github/dependabot.yml:6]: @adrian-livefront So is this `interval` is how often its checking for things like vulnerabilities? Do we know what the interval is right now?

Comment (byencho) [reply .github/dependabot.yml:6]: Interesting ok. I would think we might want this to be `daily`. Everytime there is a problem with `nokogiri` I've noticed that both the Livefront repositories and the completely separate SmartThings ones seem to get notified on the same day. That sort of suggests to me they might already be on a daily schedule.

File: .github/dependabot.yml [hunk 2]
```diff
@@ -6,4 +6,4 @@ updates:
       interval: "weekly"
     # Add/Remove reviewers here
     reviewers:
-      - "dperez37"
\ No newline at end of file
+      - "dperez37"
```
[the remark this answers is not in the document]
Comment (byencho) [reply .github/dependabot.yml:9]: I could go either way then. As long as it ends up as a single commit in the end with the original commit message.

**Review by byencho — APPROVED** [review]
Looks good. Thanks for fixing this!

**Pull request merged**